core/src/main/java/org/apache/struts2/dispatcher/StringObjectEntry.java

```diff
@@ -0,0 +1,46 @@
+package org.apache.struts2.dispatcher;
+
+import java.util.Map.Entry;
+
+abstract class StringObjectEntry implements Entry<String, Object> {
+    private String key;
+    private Object value;
+
+    public StringObjectEntry(final String key, final Object value) {
```

Comment:
```suggestion
    StringObjectEntry(final String key, final Object value) {
```

see sonar rule java:S5993